modules/va_forms/app/workers/va_forms/form_reloader.rb

```diff
@@ -44,6 +43,7 @@ def load_page(current_page: 0)
       next_button = doc.css('input[name=Next10]')
       last_page = next_button.first.attributes['disabled'].present?
       parse_page(doc)
+      current_page += 1
```

Comment:
Indexing on `vaforms/search_action.asp` starts at `0`. The way this was written with the increment at the top and the `unless current_page == 1` would cause page 2 to be skipped. It would go -> Page 1 (`unless current_page == 1` would omit the parameters, so `current_page` of 1 results in page 1) -> Page 3 (when `current_page` is 2) -> ...

Reply:
Are you counting down or up? Wouldn't this technically never stop looping if the current page was never set to zero?